po/de.po

```diff
@@ -367,70 +367,90 @@ msgid "Isolate volumes, devices and trash windows from file manager"
 msgstr "Datenträger, Geräte und Papierkorbfenster vom Dateimanager isolieren"
 
 #: Settings.ui.h:57
+msgid "Dance urgent applications"
```

Comment:
I translated _Dance_ (en) as _Wiggle_ (en) as if the original string would be “_Wiggle urgent applications_”, because a literal translation of _Dance_ would sound weird in German.

Reply:
We should probably fix the upstream code too, please do a MR changing the string :)

Reply:
https://github.com/micheleg/dash-to-dock/pull/1979